internal/process/deprovisioning/check_kyma_resource_deleted_step.go

```diff
@@ -60,7 +62,7 @@ func (step *CheckKymaResourceDeletedStep) Run(operation internal.Operation, logg
 
 	if err == nil {
 		logger.Infof("Kyma resource still exists")
-		return step.operationManager.RetryOperationWithoutFail(operation, step.Name(), "Kyma resource still exists", 15*time.Second, 30*time.Minute, logger)
+		return step.operationManager.RetryOperationWithoutFail(operation, step.Name(), "Kyma resource still exists", 15*time.Second, step.kymaResourceDeletionTimeout, logger)
```

Comment:
let's decrease the interval when the timeout is so short. Instead of having 15 sec let's have, for example 5 sec